From a3c068471000d99c8c8f5027c488f0ddae9b4287 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 16 Oct 2025 18:25:22 +0200
Subject: [PATCH] arm64: dts: st: remove GICV2M for STM32MP23

On STM32MP23 SoC, there is no PCIe, and then no GICV2M
Remove v2m node from interrupt-controller node.

Change-Id: I685c12ec50091b9a47ee45b0e87b582b6f0073bb
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/500498
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp231.dtsi | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp231.dtsi b/arch/arm64/boot/dts/st/stm32mp231.dtsi
index 3182cac784d2..dd58bd3f3c77 100644
--- a/arch/arm64/boot/dts/st/stm32mp231.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp231.dtsi
@@ -174,16 +174,6 @@ intc: interrupt-controller@4ac00000 {
 		      <0x0 0x4ac40000 0x0 0x2000>,
 		      <0x0 0x4ac60000 0x0 0x2000>;
 		interrupts = <GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(1) | IRQ_TYPE_LEVEL_LOW)>;
-
-		#address-cells = <2>;
-		#size-cells = <2>;
-		ranges;
-
-		v2m0: v2m@48090000 {
-			compatible = "arm,gic-v2m-frame";
-			reg = <0x0 0x48090000 0x0 0x1000>;
-			msi-controller;
-		};
 	};
 
 	d1_pd: power-domain-d1 {
-- 
2.53.0

